Automattic/jetpack
# AAG: Dashboard: improve perf of the AK dashboard card.

#### Changes proposed in this Pull Request:

﻿- Fetch the number of spam comments from a local option instead of making a call to the Akismet API.
- Display a better message when there are no spam comments blocked yet.

While this removes this call to the Akismet API, it also changes the data we get when we're using `getAkismetData`. The API returns a lot of data around comments, but we've really only been using the total (`all`) number of spam comments until now, which is why I think it would be fine to simplify things here. I'd be happy to have a second opinion on that though.

#### Jetpack product discussion

* pZbK5-6oF-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site that's connected to wpcom, but without a plan, and without the Akismet plugin active.
* Purchase a plan, and visit Jetpack > Dashboard again.
* Install and activate the Akismet plugin.
* Visit Jetpack > Dashboard again.
* Bump the number of spam comments saved: `jetpack docker wp option update akismet_spam_count 9822222`
* Visit Jetpack > Dashboard again.

Each time you should see a different message in the Akismet card.
